src/main/java/entity_gateway/EventEntityManager.java

```diff
@@ -0,0 +1,43 @@
+package main.java.entity_gateway;
+
+import main.java.entity.Event;
+import main.java.use_case.CalendarEventModel;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class EventEntityManager implements CalendarManager{
+    private final ArrayList<Event> eventList;
```

Comment:
Is EventEntityManager a database manager? Like is it supposed to interact with the database? If so, I don't think it should be exposed to the Event class coz it is considered a class from the outermost layer. Currently, it creates a new Event instance and stores it but I think the storage should be independent of the Event class.

Reply:
This is a point I was curious about, too. Just from the Clean Architecture diagrams, it seems like Entity Gateways interact with both Entities and the Database. I believe this is so that you can uniformly ask the gateway to give you the entity (in our case the read-only form of the entity) without caring about whether said entity was pulled from a Java class or an external file (the db).
I think the point of consideration to make is creating an actual storage another class and then an actual Gateway class which manages whether to pull from cache or db. I'm leaving this as-is for now to give @TheXanMan321 some work :)